webknossos-datastore/conf/com.scalableminds.webknossos.datastore.routes

```diff
@@ -6,6 +6,7 @@ GET           /health
 
 # Read image data
 POST          /datasets/:organizationName/:dataSetName/layers/:dataLayerName/data                                                             @com.scalableminds.webknossos.datastore.controllers.BinaryDataController.requestViaWebKnossos(token: Option[String], organizationName: String, dataSetName: String, dataLayerName: String)
+POST          /datasets/:organizationName/:dataSetName/layers/:dataLayerName/dataCuboid                                                       @com.scalableminds.webknossos.datastore.controllers.BinaryDataController.requestRawCuboidPost(token: Option[String], organizationName: String, dataSetName: String, dataLayerName: String)
```

Comment:
The route name is confusing for me since, by reading it, I would expect to be able to upload data with this route.